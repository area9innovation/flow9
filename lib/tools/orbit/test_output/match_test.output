Flow compiler (3rd generation)

Processing 'orbit' on http server

Testing conditional pattern matching
n = 5
TODO: Interpret OrRule: 5 ⇒ println("n is not greater than zero")
n is greater than zero
Result: "n is greater than zero"
\n==== EXPECTED OUTPUT DIFF ====
7d6
< TODO: Interpret OrRule: 5 ⇒ println("n is not greater than zero")
10d8
< \n==== EXPECTED OUTPUT DIFF ====
