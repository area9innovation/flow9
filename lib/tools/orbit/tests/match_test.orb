// Simple test for match with condition

println("Testing conditional pattern matching");

// Set up the test value
let n = 5;

println("n = " + n);

// Try a simple match with one condition
n is (
  x => println("n is greater than zero") if x > 0;
  
  x => println("n is not greater than zero")
)