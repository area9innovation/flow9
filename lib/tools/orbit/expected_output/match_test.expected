Testing conditional pattern matching
n = 5
n is greater than zero
Result: "n is greater than zero"
